.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow.groovy

```diff
@@ -23,7 +23,7 @@ import PostcommitJobBuilder
 // This job runs the suite of ValidatesRunner tests against the Dataflow
 // runner.
 PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Dataflow',
-    'Run Dataflow ValidatesRunner', 'Google Cloud Dataflow Runner ValidatesRunner Tests', this) {
+    'Run Dataflow ValidatesRunner', 'Google Cloud Dataflow Runner ValidatesRunner Tests (streaming/batch auto)', this) {
```

Comment:
Why is this also streaming? What does 'auto' mean?

Reply:
When `--streaming=true` is not specified, this happens:

https://github.com/apache/beam/blob/996c8d2ed77375061c90417cf4018fde604e364a/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java#L869-L871

So the unmodified `ValidatesRunner` test suite uses batch mode when possible and streaming when necessary. We expect essentially all users to use the unmodified form and let it "just work". But very few of our `ValidatesRunner` tests have unbounded PCollections so streaming is largely untested in this mode.

It may be overkill to change this title. I got worried it would be confused for a "batch only" mode, which it is not.